feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html

```diff
@@ -60,6 +60,15 @@
   >
     <ng-container *ngIf="customer$ | async; else showCustomerSelection">
       <cx-customer-emulation *ngIf="notCollapsed"></cx-customer-emulation>
+      <ng-container *cxFeatureLevel="'6.1'">
```

Comment:
Please add a TODO command to remove the feature flag associated with the Jira ticket number.
e.g.
<!-- TODO:(CXSPA-xxxx) for next major release remove feature level -->

Reply:
done